conf.py

```diff
@@ -98,10 +105,14 @@
 extensions = [
     # Parse Python docstrings (autodoc, automodule, autoattribute directives)
     'sphinx.ext.autodoc' if odoo_dir_in_path else 'autodoc_placeholder',
+    'autodoc_field',
 
     # Link sources in other projects (used to build the reference doc)
     'sphinx.ext.intersphinx',
 
+    # Embed Graphviz graphs
+    'sphinx.ext.graphviz',
+
```

Comment:
You are asking me to do changes. So yes, this is proof that it is better to have the source code available instead of a static picture. Now anyone can edit it without needing to redo everything.

Reply:
I agree with @william-andre ; if people have a warning when generating the doc and decide they don't care, it's their responsability. Having it directly in the code (and being generated automatically with the rest) makes the modifications of the graph way easier for everyone.

Reply:
Using graphviz:
* Enables proper VCS:
  * track changes more easily
  * can collaborate on the same document, everyone has the code
  * conflict management
* No blob in the repo, lighter `.git`
* No need to restart everything during reviews

Using static file:
* Shorter compile time
* No need to install graphviz

Installing graphviz, and the time to compile the graph isn't long.
If you didn't install graphviz, you still get the document generated but without the graphs, that is totally fine. Exactly like compiling without the `odoo` repo.

Reply:
SVG is a lot less easy to manipulate though